package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "develop": "gatsby develop",
+    "dev": "gatsby develop",
```

Comment:
should we keep only one of those?

also, need to update contributing guide probably (in content/docs/user-guide...)

Reply:
Removing the other one would throw off devs used to using `yarn run develop` for no real benefit, but it doesn't matter much to me so I could go either way.

Good point about the contributing guide, but if we keep the prior invocation it'd be fine as is with this just being some free syntactic sugar for devs who like to minimize their keystrokes.

What the docs are changed to, if at all, depends on what syntax we keep; because of this, I'll wait for some feedback.
I'd personally recommend to keep both commands and add a small aside in the docs mentioning that this is possible: this implementation lets people who want the feature know it's there, it won't upset anyone using the prior behavior, and it confers no performance drawback.

Reply:
Sounds good, @rogermparent . Let's keep both.

Reply:
I'd go for keeping only dev and updating the contrib guide. But I see this is merged.